frontend/src/APIClients/mutations/EntityMutations.ts

```diff
@@ -0,0 +1,33 @@
+import { gql } from "@apollo/client";
```

Comment:
Ehh I think we're reaching the point that we can start removing the entity stuff... or at the very least not touch it because it's adding unnecessary dev work

Reply:
Yeah going forward probably not necessary. I wanted to apply the logic to Entities here for my understanding since I'm not able to test with Course components :))